packages/protocol/contracts/layer1/core/impl/Inbox.sol

```diff
@@ -164,18 +147,16 @@ contract Inbox is IInbox, IForcedInclusionStore, EssentialContract {
     /// @notice Initializes the Inbox contract
     /// @param _config Configuration struct containing all constructor parameters
     constructor(IInbox.Config memory _config) {
-        require(_config.checkpointStore != address(0), ZERO_ADDRESS());
+        require(_config.signalService != address(0), ZERO_ADDRESS());
         require(_config.ringBufferSize != 0, RingBufferSizeZero());
 
         _codec = _config.codec;
         _bondToken = IERC20(_config.bondToken);
```

Comment:
Shall we remove this `IERC20 internal immutable _bondToken;` here? Looks like it's not used in the inbox anymore.